addons/website_forum/controllers/website_forum.py

```diff
@@ -659,11 +659,6 @@ def open_partner(self, forum, partner_id=0, **post):
     # Profile
     # -----------------------------------
 
-    @http.route(['/forum/user/<int:user_id>'], type='http', auth="public", website=True)
```

Comment:
Can probably be put in a ``legacy.py`` file so that URLs are not broken (not really costly and allows to keep old urls alive)